aspnetcore/data/ef-mvc/concurrency.md

```diff
@@ -17,7 +17,7 @@ uid: data/ef-mvc/concurrency
 
 By [Tom Dykstra](https://github.com/tdykstra) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-The Contoso University sample web application demonstrates how to create ASP.NET Core 1.1 MVC web applications using Entity Framework Core 1.1 and Visual Studio 2017. For information about the tutorial series, see [the first tutorial in the series](intro.md).
+The Contoso University sample web application demonstrates how to create ASP.NET Core MVC web applications using Entity Framework Core and Visual Studio. For information about the tutorial series, see [the first tutorial in the series](intro.md).
```

Comment:
Should have done that to begin with, but given short time I have left I'm inclined to leave that for the next pass through these tutorials.